### PR TITLE
Updated README to use MPE

### PR DESCRIPTION
**Pull request by raamb**
#12 
Updating the README to use the correct daemon config variables to support MPE

@vsbogd please review the changes related to etcd and if the description needs to be elaborated on. Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,21 +74,51 @@ $ python image_classification_service
     "PASSTHROUGH_ENDPOINT": "",
     "POLL_SLEEP": "",
     "PRIVATE_KEY": "",
-    "SERVICE_TYPE": "jsonrpc",
     "SSL_CERT": "",
     "SSL_KEY": "",
-    "WIRE_ENCODING": "json"
+    "LOG": {
+        "FORMATTER": {
+        "TYPE": "TEXT"
```
Comment (vsbogd) [inline README.md]: Upper case for configuration property names is ok, but most of values should be in lowercase.
Here "TEXT" in uppercase will not work.
```suggestion
        "TYPE": "text"
```

File: README.md [hunk 2]
```diff
@@ -71,21 +74,51 @@ $ python image_classification_service
...
+        },
+        "HOOKS": [],
+        "LEVEL": "INFO",
+        "OUTPUT": {
+            "CURRENT_LINK": "./SNET-DAEMON.LOG",
+            "FILE_PATTERN": "./SNET-DAEMON.%Y%M%D.LOG",
+            "MAX_AGE_IN_SEC": 604800,
+            "ROTATION_COUNT": 0,
+            "ROTATION_TIME_IN_SEC": 86400,
+            "TYPE": "FILE"
```
Comment (vsbogd) [inline README.md]:
```suggestion
            "TYPE": "file"
```

File: README.md [hunk 3]
```diff
@@ -71,21 +74,51 @@ $ python image_classification_service
...
+        },
+        "TIMEZONE": "UTC"
+    },
+    "PAYMENT_CHANNEL_STORAGE_CLIENT": {
+        "CONNECTION_TIMEOUT": "",
+        "ENDPOINTS": ["HTTP://127.0.0.1:2379"],
+        "REQUEST_TIMEOUT": "3S"
```
Comment (vsbogd) [inline README.md]: ```3S``` will not be parsed by [time.ParseDuration](https://golang.org/src/time/format.go?s=37668:37714#L1268)
```suggestion
        "REQUEST_TIMEOUT": "3s"
```

File: README.md [hunk 4]
```diff
@@ -71,21 +74,51 @@ $ python image_classification_service
...
+    },
+    "PAYMENT_CHANNEL_STORAGE_SERVER": {
+        "ID": "STORAGE-1",
+        "SCHEME": "HTTP",
```
Comment (vsbogd) [inline README.md]:
```suggestion
        "SCHEME": "http",
```

File: README.md [hunk 5]
```diff
@@ -71,21 +74,51 @@ $ python image_classification_service
...
+        "HOST" : "127.0.0.1",
+        "CLIENT_PORT": 2379,
+        "PEER_PORT": 2380,
+        "TOKEN": "UNIQUE-TOKEN",
+        "CLUSTER": "STORAGE-1=HTTP://127.0.0.1:2380",
+        "STARTUP_TIMEOUT": "1M",
```
Comment (vsbogd) [inline README.md]:
```suggestion
        "STARTUP_TIMEOUT": "1m",
```

File: README.md [hunk 6]
```diff
@@ -71,21 +74,51 @@ $ python image_classification_service
...
     "SSL_KEY": "",
-    "WIRE_ENCODING": "json"
+    "LOG": {
+        "FORMATTER": {
+        "TYPE": "TEXT"
+        },
+        "HOOKS": [],
+        "LEVEL": "INFO",
+        "OUTPUT": {
+            "CURRENT_LINK": "./SNET-DAEMON.LOG",
```
Comment (vsbogd) [inline README.md]: I would prefer have file names in lower case
```suggestion
            "CURRENT_LINK": "./snet-daemon.log",
```

File: README.md [hunk 7]
```diff
@@ -71,21 +74,51 @@ $ python image_classification_service
...
+            "FILE_PATTERN": "./SNET-DAEMON.%Y%M%D.LOG",
```
Comment (vsbogd) [inline README.md]: Date symbol case matters
```suggestion
            "FILE_PATTERN": "./snet-daemon.%Y%m%d.log",
```

File: README.md [hunk 8]
```diff
@@ -71,21 +74,51 @@ $ python image_classification_service
...
+        },
+        "TIMEZONE": "UTC"
+    },
+    "PAYMENT_CHANNEL_STORAGE_CLIENT": {
+        "CONNECTION_TIMEOUT": "",
+        "ENDPOINTS": ["HTTP://127.0.0.1:2379"],
+        "REQUEST_TIMEOUT": "3S"
+    },
+    "PAYMENT_CHANNEL_STORAGE_SERVER": {
+        "ID": "STORAGE-1",
```
Comment (vsbogd) [inline README.md]:
```suggestion
        "ID": "storage-1",
```

File: README.md [hunk 9]
```diff
@@ -71,21 +74,51 @@ $ python image_classification_service
...
+        "SCHEME": "HTTP",
+        "HOST" : "127.0.0.1",
+        "CLIENT_PORT": 2379,
+        "PEER_PORT": 2380,
+        "TOKEN": "UNIQUE-TOKEN",
+        "CLUSTER": "STORAGE-1=HTTP://127.0.0.1:2380",
```
Comment (vsbogd) [inline README.md]:
```suggestion
        "CLUSTER": "STORAGE-1=http://127.0.0.1:2380",
```

File: README.md [hunk 10]
```diff
@@ -71,21 +74,51 @@ $ python image_classification_service
...
+        "STARTUP_TIMEOUT": "1M",
+        "DATA_DIR": "STORAGE-DATA-DIR-1.ETCD",
```
Comment (vsbogd) [inline README.md]:
```suggestion
        "DATA_DIR": "storage-data-dir-1.etcd",
```

File: README.md [hunk 11]
```diff
@@ -101,10 +134,10 @@ true)
 * POLL_SLEEP: time between checking blockchain state (defaults to 5s)
```
Comment (vsbogd) [inline README.md]: POLL_SLEEP is not used as well.

File: README.md [hunk 12]
```diff
@@ -71,21 +74,51 @@ $ python image_classification_service
     "PASSTHROUGH_ENDPOINT": "",
     "POLL_SLEEP": "",
```
Comment (vsbogd) [inline README.md]: POLL_SLEEP and global LOG_LEVEL are not used anymore

File: README.md (hunk 1 above)
Comment (raamb) [reply README.md]: Done

Comment (raamb) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (raamb) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (raamb) [reply README.md]: Done

File: README.md (hunk 4 above)
Comment (raamb) [reply README.md]: Done

File: README.md (hunk 5 above)
Comment (raamb) [reply README.md]: Done

File: README.md (hunk 6 above)
Comment (raamb) [reply README.md]: Done

File: README.md (hunk 7 above)
Comment (raamb) [reply README.md]: Done

File: README.md (hunk 9 above)
Comment (raamb) [reply README.md]: Done

File: README.md (hunk 10 above)
Comment (raamb) [reply README.md]: Done

File: README.md (hunk 11 above)
Comment (raamb) [reply README.md]: Done

File: README.md (hunk 12 above)
Comment (raamb) [reply README.md]: Done

**Pull request merged**